app/api/logs/block-height/route.ts

```diff
@@ -48,11 +50,8 @@ type LogEntry = {
   timestamp: string;
 };
 
-const CHUNK_SIZE = 6 * 60 * 60 * 1000; // 6 hours or 21600000 ms
 const COMMIT_BLOCK_FINISH_EVENT_TYPE = 'cosmic-swingset-commit-block-finish';
-const LOG_ENTRIES_ENDPOINT = 'https://logging.googleapis.com/v2/entries:list';
-const MAXIMUM_DAYS = 30;
-const MILLI_SECONDS_PER_DAY = 24 * 60 * 60 * 1000; // 86400000 ms
+const POD_FILTER = `"resourceLabels" @@ '$.pod_name == "follower-0"' OR "resourceLabels" @@ '$.pod_name == "fork1-0"' OR "resourceLabels" @@ '$.pod_name == "validator-primary-0"'`;
```

Comment:
What is this @@ symbol?

Reply:
This is postgresql specifc syntax for applying conditions on json